### PR TITLE
Fix milestone workflow

### PR DESCRIPTION
**Pull request by wendigo**
```
+ ./mvnw -v
Unrecognized VM option 'G1NumCollectionsKeepPinned=10000000'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

### CODE REVIEW

File: .github/workflows/milestone.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - uses: ./.github/actions/setup
+        timeout-minutes: 10
+        with:
+          java-version: 22
```
Comment (findepi) [inline .github/workflows/milestone.yml:21]: before the change, what was it using?

**Review by findepi — APPROVED** [review]

File: .github/workflows/milestone.yml (hunk 1 above)
Comment (wendigo) [reply .github/workflows/milestone.yml:21]: I believe that one that came by default with a worker. Let me check

Comment (wendigo) [reply .github/workflows/milestone.yml:21]: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#java

JDK 17

**Comment by wendigo** [comment]
Passed:

Screenshot 2024-05-17 at 22 59 59

**Pull request merged**